### PR TITLE
Document the K3s config file

### PR DESCRIPTION
**Pull request by catherineluse**
This PR addresses this issue https://github.com/rancher/k3s/issues/2057

### CODE REVIEW

File: content/k3s/latest/en/installation/install-options/_index.md [hunk 1]
```diff
@@ -71,3 +73,37 @@ For details on configuring the K3s server, refer to the [server configuration re
...
+### Configuration File
+
+In addition to configuring K3s with environment variables and CLI arguments, K3s can also use a config file.
+
+By default, values present in a YAML file located at `/etc/rancher/k3s/flags.conf` will be used on install.
+
+An example of a basic `server` config file is below:
+
+```yaml
+# /etc/rancher/rke2/flags.conf
```
Comment (erikwilson) [inline content/k3s/latest/en/installation/install-options/_index.md:86]: Might want to remove this comment

**Review by erikwilson — APPROVED** [review]

File: content/k3s/latest/en/installation/install-options/_index.md [hunk 2]
```diff
@@ -71,3 +73,37 @@ For details on configuring the K3s server, refer to the [server configuration re
 ### Registration Options for the K3s Agent
 
 For details on configuring the K3s agent, refer to the [agent configuration reference.]({{<baseurl>}}/k3s/latest/en/installation/install-options/agent-config)
+
+### Configuration File
+
+In addition to configuring K3s with environment variables and CLI arguments, K3s can also use a config file.
+
+By default, values present in a YAML file located at `/etc/rancher/k3s/flags.conf` will be used on install.
```
Comment (ShylajaDevadiga) [inline content/k3s/latest/en/installation/install-options/_index.md:81]: Replace flags.conf to config.yaml

File: content/k3s/latest/en/installation/install-options/_index.md (hunk 1 above)
Comment (catherineluse) [reply content/k3s/latest/en/installation/install-options/_index.md:86]: Removed the comment

File: content/k3s/latest/en/installation/install-options/_index.md (hunk 2 above)
Comment (catherineluse) [reply content/k3s/latest/en/installation/install-options/_index.md:81]: replaced

**Review by davidnuzik — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**